docs/buildah.md

```diff
@@ -12,19 +16,54 @@ The buildah package provides a command line tool which can be used to:
     * Use the updated contents of a container's root filesystem as a filesystem layer to create a new image.
     * Delete a working container or an image.
 
+## OPTIONS
+
+**--root** **value**
+
+Storage root dir (default: "/var/lib/containers/storage")
+
+**--runroot** **value**
+
+Storage state dir (default: "/var/run/containers/storage")
+
+**--storage-driver** **value**
+
+Storage driver
+
+**--storage-opt** **value**
+
+Storage driver option
+
+**--debug**
+
+Print debugging information
+
+**--help, -h**
+
+Show help
+
+**--version, -v**
+
+Print the version
+
+
 ## SEE ALSO
-| Command               | Description |
-| --------------------- | --------------------------------------------------- |
-| buildah-add(1)        | Add the contents of a file, URL, or a directory to the container. |
-| buildah-bud(1)        | Build an image using instructions from Dockerfiles. |
-| buildah-commit(1)     | Create an image from a working container. |
-| buildah-config(1)     | Update image configuration settings. |
-| buildah-containers(1) | List the working containers and their base images. |
-| buildah-copy(1)       | Copies the contents of a file, URL, or directory into a container's working directory. |
+
+| Command               | Description                                                                                          |
+| --------------------- | ---------------------------------------------------                                                  |
+|                       |                                                                                                      |
+| buildah-add(1)        | Add the contents of a file, URL, or a directory to the container.                                    |
+| buildah-bud(1)        | Build an image using instructions from Dockerfiles.                                                  |
```

Comment:
Not sure if we want to go there or not, but perhaps instead of "Dockerfiles" maybe "image creation scripts such as Dockerfiles" or just "image creation scripts".  I'm just wondering if we should try to pull Buildah further away from Docker, or if at this point "Dockerfile" is like "Kleenex".